arm_navigation_msgs/RobotState[] robot_states

---

arm_navigation_msgs/ArmNavigationErrorCodes[] error_codes
